shell/app/common/utils/__tests__/query-string.test.tsx

```diff
@@ -47,13 +45,19 @@ describe('query-string', () => {
     expect(mergeSearch({ orgName: 'erda' }, false, true)).toEqual({ orgName: 'erda' });
   });
   it('updateSearch', () => {
-    const history = getConfig('history');
+    const goTo = jest.fn();
+    Object.defineProperty(GoTo, 'goTo', {
+      value: goTo,
+    });
     updateSearch({ orgName: 'erda' });
-    expect(history.location.search).toContain('orgName=erda');
+    expect(goTo).toHaveBeenCalled();
```

Comment:
This assert can not test the updateSearch result.

Reply:
goTo is called inside updateSearch, so just need to make sure that goTo is called correctly